src/components/CreateProfile/CreateProfileTextArea/CreateProfileTextArea.jsx

```diff
@@ -0,0 +1,15 @@
+'use client'
+import React from 'react'
+import styles from './CreateProfileTextarea.module.scss'
+
+const CreateProfileTextArea = ({ placeholder }) => {
```

Comment:
**nitpick:** Same here - why `CreateProfileTextArea` if it's a generic component. We are also missing props interface here